pkg/pod/pods.go

```diff
@@ -68,33 +69,65 @@ const (
 	AbortedPrepare = "aborted prepare"
 	Prepared       = "prepared"
 	Running        = "running"
-	Deleting       = "deleting" // This covers pod.isExitedDeleting and pod.isDeleting.
-	Exited         = "exited"   // This covers pod.isExited and pod.isExitedGarbage.
+	Deleting       = "deleting"
+	ExitedDeleting = "exited deleting"
```

Comment:
The `ExitedDeleting` state currently is not documented. If we export this now, we should also update `Documentation/devel/pod-lifecycle.md`.